common/config.gni

```diff
@@ -22,6 +22,9 @@ declare_args() {
 
   # Whether to use a prebuilt Dart SDK instead of building one.
   flutter_prebuilt_dart_sdk = false
+
+  # The android cpu used to generate windows artifacts.
+  android_cpu = "arm"
```

Comment:
This doesn't seem quite right. I need some time to investigate.

Reply:
It seems like this workaround would be a continuation of this hack:

https://github.com/flutter/engine/blob/4b32f2ea24622011729face61e91bf09e6178de0/tools/gn#L289

Instead of going further in this direction, we should try to remove the hack. (Added/expanded on here https://github.com/flutter/engine/pull/3883 cc @rmacnak-google)

Reply:
Following https://github.com/flutter/engine/pull/37125, I think `android_cpu` is no longer needed, and `target_cpu` can be used. But I haven't actually tried it myself ;-)